edx/frontend-app-course-authoring
# feat: teams settings editor [BD-38] [TNL-8453] [BB-4843]

Adds a new editor for the team sets, completing all the apps for pages and resources.

**JIRA tickets**:  TNL-8101 [TNL-8453](https://openedx.atlassian.net/browse/TNL-8453)

**Screenshots**: 


https://user-images.githubusercontent.com/118837/132700674-ee7c8d45-8402-459d-aa2e-c3ce15139f0a.mp4



**Testing instructions**:

1. For a course that doesn't have teams configured, visit http://localhost:2001/course/course-v1:edX+DemoX+Demo_Course/pages-and-resources/teams/settings
2. Add a new team set
3. Check that the validation works properly
4. Check that you can add teams
5. Check that you can delete teams
6. Check that you can edit teams
7. Check that the teams show up in the LMS at http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/


**Reviewers**
